src/main/java/commands/ListCardCommand.java

```diff
@@ -0,0 +1,47 @@
+package commands;
+
+import access.Access;
+import manager.card.Card;
+import manager.chapter.CardList;
+import storage.Storage;
+import ui.Ui;
+
+import java.util.ArrayList;
+
+import static common.Messages.CARD;
+
+public class ListCardCommand extends ListCommand {
+    public static final String COMMAND_WORD = "list";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
```

Comment:
Perhaps can change flashcards to %ss (extra s for the word) in ListCommand and when you use this message, Stringformat with the type Messages.CARD